chia/full_node/full_node_api.py

```diff
@@ -863,12 +860,15 @@ async def declare_proof_of_space(
                     while not curr_l_tb.is_transaction_block:
                         curr_l_tb = self.full_node.blockchain.block_record(curr_l_tb.prev_hash)
                     try:
-                        block = await self.full_node.mempool_manager.create_block_generator(curr_l_tb.header_hash)
-                        if block is not None:
-                            block_generator, aggregate_signature, additions, removals = block
+                        new_block_gen = await self.full_node.mempool_manager.create_block_generator(
+                            curr_l_tb.header_hash
+                        )
                     except Exception as e:
                         self.log.error(f"Traceback: {traceback.format_exc()}")
                         self.full_node.log.error(f"Error making spend bundle {e} peak: {peak}")
+                        new_block_gen = None
```

Comment:
Then we don't have to set it to None explicitly, since it's the default

Reply:
I specifically made this change so that `mypy` can catch places where it's not deliberately set. it allows us to pretend that the variable is immutable too.